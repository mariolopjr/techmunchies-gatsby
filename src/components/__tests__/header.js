import React from "react"
import renderer from "react-test-renderer"

import Header from "../header"

const links = [`home`, `projects`, `blog`]

describe("Header", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<Header siteName="techmunchies" />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
