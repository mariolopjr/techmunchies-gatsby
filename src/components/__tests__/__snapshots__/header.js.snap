// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<nav
  className="css-1on83g2-Nav e1a59lgm0"
>
  <div
    className="css-1k2ouqc-NavBrand e1a59lgm5"
  >
    <a
      className="css-wbn05c-NavBrandLink e1a59lgm6"
      href="/"
    >
      techmunchies
    </a>
    <a
      aria-expanded={false}
      aria-label="menu"
      className="css-1lsnd9l-MenuButton e1a59lgm3"
      onClick={[Function]}
      role="button"
    >
      <span
        aria-hidden="true"
        className="css-w3xjge-MenuButtonBar e1a59lgm4"
      />
      <span
        aria-hidden="true"
        className="css-w3xjge-MenuButtonBar e1a59lgm4"
      />
      <span
        aria-hidden="true"
        className="css-w3xjge-MenuButtonBar e1a59lgm4"
      />
    </a>
  </div>
  <div
    className="css-klxq16-NavContainer e1a59lgm1"
  >
    <div
      className="css-13wg8wn-NavContainerEnd e1a59lgm2"
    >
      <div
        className="css-y16p2q-NavLinkContainer e1a59lgm7"
      >
        <a
          className="css-f83ofw-NavLink e1a59lgm8"
          href="/"
        >
          home
        </a>
        <a
          className="css-f83ofw-NavLink e1a59lgm8"
          href="/projects/"
        >
          projects
        </a>
        <a
          className="css-f83ofw-NavLink e1a59lgm8"
          href="/blog/"
        >
          blog
        </a>
      </div>
    </div>
  </div>
</nav>
`;
